src/azure-cli/azure/cli/command_modules/iot/tests/latest/test_iot_commands.py

```diff
@@ -404,7 +407,7 @@ def test_identity_hub(self, resource_group, resource_group_location, storage_acc
         location = resource_group_location
 
         private_endpoint_type = 'Microsoft.Devices/IoTHubs'
-        identity_hub = 'identity-test-hub-cli-1'
+        identity_hub = 'identity-test-hub-cli-2'
```

Comment:
May I ask why `identity_hub` needs to be modified?
If it's not a reusable value, how about changing it to a random value?

Reply:
@zhoxing-ms - sure, this value was only changed due to a previous test run failing to delete the resource.

I've gone ahead and changed the value to a randomly generated one and updated the test recording.